ext/PlotsGRExt/gr.jl

```diff
@@ -751,8 +752,8 @@ function gr_get_ticks_size(ticks, rot)
     w, h
 end
 
-function labelfunc(scale::Symbol, backend::GRBackend)
-    texfunc = labelfunc_tex(scale)
+function Plots.labelfunc(scale::Symbol, backend::GRBackend)
+    texfunc = Plots.labelfunc_tex(scale)
```

Comment:
In general I'd prefer to not have these module declarations inline. It should be visible from the top of the file which functions belong to a different module by 
- `include`ing functions that get extended
- `using` functions, that are used

Reply:
Sure let's keep it this way then